src/Microsoft.Health.Dicom.Functions/Indexing/ReindexDurableFunction.Orchestration.cs

```diff
@@ -0,0 +1,122 @@
+// -------------------------------------------------------------------------------------------------
```

Comment:
There look to be a number of pending comments from my last iteration that still need review